src/content/docs/mlops/bring-your-own/getting-started-byo.mdx

```diff
@@ -85,7 +85,7 @@ Done! To check your application go to **[one.newrelic.com](https://one.newrelic.
 
 ## Examples [#examples]
 We created these notebooks in Google colab so you can try them out:
-  1. [Try out](https://github.com/newrelic-experimental/ml-performance-monitoring/blob/main/examples/XGBoost_on_Boston_housing_prices_dataset.ipynb) an XGBoost model on [Boston housing prices](https://scikit-learn.org/stable/modules/generated/sklearn.datasets.load_boston.html) dataset. Open in [colab](https://colab.research.google.com/github/newrelic-experimental/ml-performance-monitoring/blob/main/examples/XGBoost_on_Boston_housing_prices_dataset.ipynb).
+  1. [Try out](https://github.com/newrelic-experimental/ml-performance-monitoring/blob/main/examples/XGBoost_on_Boston_housing_prices_dataset.ipynb) an XGBoost model on [Boston housing prices](https://scikit-learn.org/0.15/modules/generated/sklearn.datasets.load_boston.html) dataset. Open in [colab](https://colab.research.google.com/github/newrelic-experimental/ml-performance-monitoring/blob/main/examples/XGBoost_on_Boston_housing_prices_dataset.ipynb).
```

Comment:
Not sure why this change was introduced, I didn't do it from my side 🤷‍♂️

Reply:
@jsubirat I will roll this change since the MLOps team is going to create a separate Jira to correct this.